src/PSRule/Host/HostHelper.cs

```diff
@@ -97,7 +103,12 @@ internal static void ImportResource(Source[] source, RunspaceContext context)
             if (source == null || source.Length == 0)
                 return;
 
-            Import(ReadYamlObjects(source, context), context);
+            var results = new List<ILanguageBlock>();
+
+            results.AddRange(ReadYamlObjects(source, context));
+            results.AddRange(ReadJsonObjects(source, context));
```

Comment:
Wasn't sure if this is what we should do, but basically combining YAML and JSON result blocks. Did the same for baselines.

Reply:
I think that is fine.